<HTML>
<HEAD>
<TITLE>Poster List</TITLE>
</HEAD>
<BODY BGCOLOR="#ffffff">

<UL>
	<FONT FACE="HELVETICA" SIZE="5">Poster List (140)</FONT>
	<P>

<FONT FACE="HELVETICA" SIZE="4">
<P>

<OL>

<LI> Olivier Andrieu, Dominique Anxolab&eacute;h&egrave;re, Hadi Quesneville<BR>
     Detecting transposable elements with less a priori<BR>  
     &nbsp;<BR>
</LI>

<LI> Andreas D. Baxevanis, Izabela Makalowska, Kenneth Trout, Qien Zhou, ZhengZheng Zhou,
     Jaime Stein, Edward R. Dougherty, Seungchan Kim, Paul S. Meltzer, Yidong Chen,
     Michael L. Bittner, andJeffrey M. Trent<BR>
     An Integrated Approach for the Storage, Exploration, and Analysis of Microarray Gene
     Expression Data.<BR>  
     &nbsp;<BR>
</LI>

<LI> Osman B. Bayazit, Guang Song, Nancy M. Amato<BR>
     Ligand Binding with OBPRM and Haptic Input.<BR>
     &nbsp;<BR>
</LI>

<LI> Anne Bergeron, Sylvie Hamel<BR>
     A Vector Algorithm for Approximate String Matching with Arbitrary Weighted Distances.<BR>
     &nbsp;<BR>
</LI>

<LI> Pedro Bernaola-Galv&aacute;n, Pedro Carpena, Ram&oacute;n Rom&aacute;n-Rold&aacute;n,
     Jos&eacute; L. Oliver<BR>
     Mapping isochores by entropic segmentation of long genome sequences.<BR>
     &nbsp;<BR>
</LI>

<LI> Peter J.  Bickel, Alexander N. Glazer, Katherina J. Kechris, Phil C. Spector, Gary J. Wedemayer<BR>
     Finding "Important" Sites in Protein Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Thomas C. Bishop, Oleksandr O.Zhmudsky<BR>
     Information transmission along DNA<BR>  
     &nbsp;<BR>
</LI>

<LI> Jacek Blazewicz, Piotr Formanowicz, Marcin Jaroszewski, Marta Kasprzak,
      Wojciech T. Markiewicz<BR>
     Simplified partial digest method for DNA restriction maps construction<BR>  
     &nbsp;<BR>
</LI>

<LI> Michael B. Bober, Paul Delmar, Suzanne T. Szak, Ulas Karaoz, Jody A. White,
     Jim Hudson, Mark Boguski, Gordon Tomaselli, Raimond L. Winslow<BR>
     Gene expression in human heart failure: Microarray analysis of dilated cardiomyopathy<BR>  
     &nbsp;<BR>
</LI>

<LI> Michael B. Bober, Tugba Onal Suzek, Kevin Wiehe, Raimond L. Winslow<BR>
     The human cardiac gene expression knowledgebase.<BR>
     &nbsp;<BR>
</LI>

<LI> Robert J. Boissy, Douglas A. Bell<BR>
     Structured query fragment analysis.<BR>
     &nbsp;<BR>
</LI>

<LI> Robert J. Boissy, Andrew E. Dellinger, Douglas A. Bell<BR>
     A structured query fragment analysis database<BR>  
     &nbsp;<BR>
</LI>

<LI> Wesley D. Bonds<BR>
     Bioinformatics as an inexpensive vector for introducing modern biological concepts into the american high school curriculum<BR>  
     &nbsp;<BR>
</LI>

<LI> Guillaume Bourque, Pavel Pevzner<BR>
     Heuristic for Genome Rearrangements in Multi-Chromosomal Species<BR>  
     &nbsp;<BR>
</LI>

<LI> Carlos A. Del Carpio, Atsushi Yoshimori<BR>
     Flexible protein docking oriented to the inference of structural characteristics
     involved in molecular recognition<BR>    
     &nbsp;<BR>
</LI>

<LI> Julie Chabalier, C&eacute;cile Capponi, Yves Quentin, Gwennaele Fichant<BR>
     A hierarchical object-relation knowledge base for identifying ABC transporters<BR>  
     &nbsp;<BR>
</LI>

<LI> Christo Christov, Boris Atanasov<BR>
     pH-dependent electrostatics of class C Beta-Lactamase (Enterobacter cloacae): Tyrosyl ionizations.<BR>
     &nbsp;<BR>
</LI>

<LI> Trees-Juen Chuang, Wen-Chuang Lin, Lan-Yang Ch'ang<BR>
     Framework annotation of the human genome by CRASA<BR>  
     &nbsp;<BR>
</LI>

<LI> Cathy Cleland, Aaron Day, Kimberly Prichard, Jean Challacombe, Thomas Brettin<BR>
     XBase: an open source gene expression database project<BR>  
     &nbsp;<BR>
</LI>

<LI> Christian Cumbaa<BR>
     Modeling Protein Secondary Structure by Combination of Dependent Experts<BR>  
     &nbsp;<BR>
</LI>

<LI> Juan B. del Cuvillo, Wellington S. Martins, Guang R. Gao, Wenwu Cui, Sun Kim<BR>
     ATGC: Another Tool for Genome Comparison<BR>  
     &nbsp;<BR>
</LI>

<LI> Andrew Damokosh, Barbara Bryant, Michael Morrissey, Xumei Zhao, Sebastian Hoersch, Greg Tucker-Kellogg, Robert Schlegel<BR>
     A Novel Heuristic Algorithm For Identifying Sets of Genetic Markers In Transcriptional Profiling Data<BR>  
     &nbsp;<BR>
</LI>

<LI> Wayne Dawson, Kenji Yamamoto<BR>
     Cross linking entropic evaluation in RNA secondary structure free energy calculations<BR>  
     &nbsp;<BR>
</LI>

<LI> Xiaotie Deng, Zimao Li, Bin Ma, Lusheng Wang<BR>
     Approximation Algorithms for Distinguishing String Selection Problems<BR>  
     &nbsp;<BR>
</LI>

<LI> Qunfeng Dong, Zhijun Wu<BR>
     A Linear-Time Algorithm for Solving the Molecular Distance Geometry Problem<BR>  
     &nbsp;<BR>
</LI>

<LI> Christine Dufraigne, Bernard Fertil, Alain Giron, Patrick J. Deschavanne<BR>
     Use of genomic signature to detect horizontal DNA transfers<BR>  
     &nbsp;<BR>
</LI>

<LI> Roland L. Dunbrack, Guoli Wang, J. Michael Sauder, Jonathan Arthur<BR>
     Benchmarks for Protein Structure Prediction<BR>  
     &nbsp;<BR>
</LI>

<LI> Lars Ebert, Rita Schatten, Bernd Korn, Annemarie Poustka<BR>
     High throughput subcloning of complete protein coding open reading frames (ORF)<BR>  
     &nbsp;<BR>
</LI>

<LI> Markus Eiglsperger, Michael Kaufmann<BR>
     Visualization of Biodegradation Pathways in the UM-BBD<BR>  
     &nbsp;<BR>
</LI>

<LI> Nadia El-Mabrouk<BR>
     Reconstructing an ancestral genome by reversals and multiple gene duplication<BR>  
     &nbsp;<BR>
</LI>

<LI> J. Mark Ettinger, Thomas Brettin<BR>
     Using Conditional Entropy to Investigate Genetic Networks<BR>  
     &nbsp;<BR>
</LI>

<LI> Patricia Evans, Todd Wareham<BR>
     Practical Algorithms for Universal DNA Primer Design:
     An Exercise in Algorithm Engineering<BR>  
     &nbsp;<BR>
</LI>

<LI> Eran Eyal, Rafael Najmanovich, Mariana Babor, Marvin Edelman, Vladimir Sobolev<BR>
     Protein side-chain flexibility upon point mutations: Deriving the upper limit of
     predictive accuracy from PDB files<BR>  
     &nbsp;<BR>
</LI>

<LI> Shu-Fen Fang, Hsien-Da Huang, Jorng-Tzong Horng<BR>
     Reconstructing Phylogenetic Trees Based on Ribosomal RNA Secondary Structures<BR>  
     &nbsp;<BR>
</LI>

<LI> Nir Friedman, Itsik Pe'er, Tal Pupko<BR>
     Branch-and-Bound Reconstruction of Ancestral Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Alberto de la Fuente, Pedro Mendes<BR>
     Arguments for measuring the metabolome in functional genomics research<BR>  
     &nbsp;<BR>
</LI>

<LI> Boris Galitsky, Sergey Shelepin<BR>
     Using correlation matrices in the comparative analysis of immunoglobulin chains<BR>  
     &nbsp;<BR>
</LI>

<LI> Gopinath Ganji, Yingfu Li, A. Jamie Cuticchia<BR>
     Computational Analysis of DNAZymes<BR>
     &nbsp;<BR>
</LI>

<LI> Xiang Gao, Kent Vander Velden, Daniel F. Voytas<BR>
     Tree-based method to identify protein functional domains: Case study of
      primer utilization by reverse transcriptase<BR>  
     &nbsp;<BR>
</LI>

<LI> Chris Gentner, Rolf D. Schmid, J&uuml;rgen Pleiss<BR>
     Enantioselectivity of Pseudomonas cepacia lipase and mutants towards
     primary alcohols: Prediction by an in silico assay<BR>  
     &nbsp;<BR>
</LI>

<LI> Artur Gieldon, Cezary Czaplewski, Rajmund Kazmierkiewicz, Rafal Slusarz,
     Marta Pasenkiewicz-Gierula, Jerzy Ciarkowski<BR>
     Molecular Modeling of Interactions of no-Peptide Antagonist OPC-31360 with
     the Human Vasopresin V2 Receptor<BR>
     &nbsp;<BR>
</LI>

<LI> Thomas G. Graeber, David Eisenberg<BR>
     Bioinformatic analysis of gene expression patterns suggests chemokine
     autocrine loops in lymphomas and leukemias<BR>  
     &nbsp;<BR>
</LI>

<LI> Jens Gramm, Rolf Niedermeier<BR>
     Evaluating an algorithm for parameterized Minimum Quartet Inconsistency<BR>  
     &nbsp;<BR>
</LI>

<LI> Debraj GuhaThakurta, Gary D. Stormo<BR>
     Identifying Target Sites for Cooperatively Binding Factors<BR>  
     &nbsp;<BR>
</LI>

<LI> Jari H&auml;kkinen, Thorsteinn R&ouml;gnvaldsson, Jim Samuelsson<BR>
     A novel computational approach to protein identification<BR>  
     &nbsp;<BR>
</LI>

<LI> Thomas Hagedorn<BR>
     Invariants for the General Model<BR>  
     &nbsp;<BR>
</LI>

<LI> Bjarni V. Halld&oacute;rsson, Jonathan S. Minden, R. Ravi<BR>
     Protein Identification by Epitope Recognition<BR>  
     &nbsp;<BR>
</LI>

<LI> Rhonda Harrison, Francois Robert, Bing Ren, John Wyrick,
     Charles DeLisi, Richard A. Young<BR>
     A computational approach to condition-based promoter characterization in
     S.c. based on genome-wide expression and binding profiles<BR>  
     &nbsp;<BR>
</LI>

<LI> Artemis G. Hatzigeorgiou<BR>
     Translation Initiation Site Prediction on full length and partial cDNA's
     using Artificial Neural Networks<BR>  
     &nbsp;<BR>
</LI>

<LI> P. Havlak, K. Durbin, X. Ma, O. Lichtarge, R. Gibbs, J. Bouck<BR>
     The Atlas Whole-Genome Assembler<BR>  
     &nbsp;<BR>
</LI>

<LI> Peter Hodges, Pinar Kondu Qi Sun<BR>
     A BioKnowledge(TM Library for Rapid Assessment of the Function of Novel Human Genes<BR>  
     &nbsp;<BR>
</LI>

<LI> Paul G. Hodor, Rich Caruana, Bruce G. Buchanan<BR>
     Rare gene expression patterns discovered by hierarchical clustering from
     microarray data<BR>  
     &nbsp;<BR>
</LI>

<LI> Michael H&ouml;rnquist, John Hertz, Mattias Wahde<BR>
     Effective dimensionality of expression data from development of rat central nervous system<BR>  
     &nbsp;<BR>
</LI>

<LI> Jorng-Tzong Horng, Cho-Chun Huang, Hsien-Da Huang ,Cheng-Yan Kao<BR>
     Mining The Putative Functional Sites Using Repetitive Elements<BR>  
     &nbsp;<BR>
</LI>

<LI> Gang Ken Hu, Brian Moldover, Tim Jatkoe, Steve Madore, David Balaban,
     Jeffrey Thomas, Yixin Wang<BR>
     Predicting Splice Variant from DNA Chip Expression Data<BR>  
     &nbsp;<BR>
</LI>

<LI> Yuanpeng Janet Huang, Roberto Tejero, Robert Powers, Norma J. Greenfield,
     Bonnie L. Dixon, G.V.T. Swapna, Gaetano T. Montelione<BR>
     Automated analysis of three-dimensional protein structures from NOESY data
     using AutoStructure<BR>  
     &nbsp;<BR>
</LI>

<LI> Wolfgang Huber, Anja von Heydebreck, Holger S&ouml;ltmann, Annemarie Poustka, Martin Vingron<BR>
     Data preprocessing and quality control for DNA array expression profiling experiments<BR>  
     &nbsp;<BR>
</LI>

<LI> Tor-Kristian Jenssen, Winston Patrick Kuo, Atul Butte, Isaac Kohane<BR>
     Possible negative effects of reference mRNA sample in cDNA microarrays<BR>  
     &nbsp;<BR>
</LI>

<LI> Howsun Jow, Magnus Rattray, David C. Hoyle, Cendrine Hudelot, Paul G. Higgs<BR>
     RNA Evolution and Molecular Phylogenetics<BR>  
     &nbsp;<BR>
</LI>

<LI> Sumie K-Abe, Nobuyuki Takahashi, Mineichi Kudo, Masaru Shimbo, Akihiro Tsutsumi<BR>
     Distribution Analysis of 5’ Splice Site-Like Sequences in human mRNA Precursors<BR>  
     &nbsp;<BR>
</LI>

<LI> Alexander A. Kanapin, Wolfgang Fleischmann, Rolf Apweiler<BR>
     Integration of biological databases: the example of InterPro<BR>  
     &nbsp;<BR>
</LI>

<LI> Nikola Kasabov, Matthias Futschik<BR>
     Evolving Fuzzy Neural Networks for Knowledge Discovery from Gene Expression
     Data - a Case Study<BR>  
     &nbsp;<BR>
</LI>

<LI> Alexander Kel, Theresa Zhang, Michael Zhang, Olga Kel-Margoulis,
     Tatyana Ivanova, Edgar Wingender<BR>
     A decision tree classifier of promoters based on composite modules of TF binding sites<BR>  
     &nbsp;<BR>
</LI>

<LI> M. Kathleen Kerr, Gary Churchill<BR>
     Statistical Inference for Gene Expression Microarray Data Analysis<BR>  
     &nbsp;<BR>
</LI>

<LI> George A. Komatsoulis, Henrik Olsen<BR>
     New computational tools for analyzing and searching gene expression data
     derived from TaqMan real-time PCR.<BR>
     &nbsp;<BR>
</LI>

<LI> Gregory V. Kryukov, Alexei V. Lobanov, Vadim N. Gladyshev<BR>
     New selenocysteine-containing proteins identified with a program that searches for
     an mRNA structural element<BR>  
     &nbsp;<BR>
</LI>

<LI> Yosef Yehuda Kuttner, Mariana Babor, Marvin Edelman, Vladimir Sobolev<BR>
     Structural commonality in protein binding sites for ATP<BR>  
     &nbsp;<BR>
</LI>

<LI> Igor Kuznetsov, Shalom Rackovsky<BR>
     Informative Descriptors of Non-Local Residue Environments in Protein Structures<BR>  
     &nbsp;<BR>
</LI>

<LI> Koji M. Kyoda, Shuichi Onami, Hiroaki Kitano<BR>
     The DBRF Method for Inferring a Gene Network from Steady-State Gene Expression Data<BR>  
     &nbsp;<BR>
</LI>

<LI> Joseph I. Landman, Haruna N. Cofer, Dmitri Mikhailov, Roberto Gomperts<BR>
     Modifying Standard Bioinformatics Analysis Tools For Use On High Performance
     Computational Platforms<BR>  
     &nbsp;<BR>
</LI>

<LI> François-Joseph Lapointe, Pierre-Alexandre Landry<BR>
     A Fast Procedure for Estimating Missing Distances in Incomplete Matrices
     Prior to Phylogenetic Analysis<BR>  
     &nbsp;<BR>
</LI>

<LI> Hal A Lewis, J Michael Sauder, Sean Buchanan<BR>
     Insights from a structural genomics approach<BR>  
     &nbsp;<BR>
</LI>

<LI> Timothy G. Lilburn, George M. Garrity<BR>
     Exploratory data analysis and the visualization of extremely large sequence data sets<BR>  
     &nbsp;<BR>
</LI>

<LI> Chaim Linhart, Ron Shamir<BR>
     The Degenerate Primer Selection Problem<BR>  
     &nbsp;<BR>
</LI>

<LI> Doron Lipson, Ziv Nevo, Ari Frank, Dolev Dotan, Zohar Yakhini<BR>
     Sequencing by Hybridization - A Simulation Study of Performance on Genomic Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Yuh-Fan Liu, Yu-Huei Liao, Ueng-Cheng Yang<BR>
     Genome-wide Motif Scanning<BR>  
     &nbsp;<BR>
</LI>

<LI> Ari L&ouml;ytynoja, Michel C. Milinkovitch<BR>
     SOAP, a Program for Cleaning Multiple Alignments from Unstable Blocks<BR>  
     &nbsp;<BR>
</LI>

<LI> Szu-Hsien Lu, Ming-Jing Hwang<BR>
     Clustering Protein Sequences by Parallel Mining Non-positional k-tuple Matches<BR>  
     &nbsp;<BR>
</LI>

<LI> S. K. Luedemann, P. J. Winn, R. Gauges, V. Lounnas, R. C. Wade<BR>
     Comparison of the dynamics of substrate access channels in three bacterial
     cytochrome P450 proteins reveals different opening mechanisms and a new functional
     role for a buried arginine<BR>  
     &nbsp;<BR>
</LI>

<LI> Donna Maglott, Maria Korab-Laskowska, Kim Pruitt<BR>
     NCBI~Rs LocusLink: data integration to facilitate navigation within and
     between several model organisms<BR>  
     &nbsp;<BR>
</LI>

<LI> Vladimir Makarenkov, Eric Bazin, Pierre Legendre<BR>
     T-REX - software for reconstructing phylogenetic trees and reticulation networks.
     The multiple regression on distance matrices as a comparative method<BR>  
     &nbsp;<BR>
</LI>

<LI> Philippe Marc, Frederic Devaux, Stephane Le Crom, Imrich Hikkel, Xiaoting Zhang,
     Scott Moye-Rowley, Claude Jacq<BR>
     Regulations of the transcriptome adaptations to cellular environment:
     the yeast multidrug resistance phenomenon<BR>  
     &nbsp;<BR>
</LI>

<LI> Leonardo de O. Martins<BR>
     fprotml: Protein Analysis Using Maximum Likelihood<BR>  
     &nbsp;<BR>
</LI>

<LI> Hiroshi Matsuno, Atsushi Doi, Rainer Drath, Satoru Miyano<BR>
     Genomic Object Net: Hybrid Petri Net for Describing Biological Systems<BR>  
     &nbsp;<BR>
</LI>

<LI> Ivan V. Matveev, Ivan B. Lobov<BR>
     Specificity of SAF-A and lamin B binding in vitro correlates with the satellite
     DNA bending state<BR>  
     &nbsp;<BR>
</LI>

<LI> Marcella A. McClure, Eric F. Donaldson, Helga Ihsle Pac, Bradley Custer,
     Aaron Juntunen, Joel Graff , Kate Spizziri, Kate McInnerney<BR>
     Identification of Retroid Agents in the Human Genome<BR>  
     &nbsp;<BR>
</LI>

<LI> Brendan McConkey, Vladimir Sobolev, Marvin Edelman<BR>
     Ligand-Protein Docking Using a Two Stage Algorithm: 1. Rapid Determination of
     Binding Site and Approximate Ligand Position<BR>  
     &nbsp;<BR>
</LI>

<LI> Luciano Milanesi, Raffaella Rizzi, Mario Falchi<BR>
     Microarray spot reading: SpotView<BR>  
     &nbsp;<BR>
</LI>

<LI> Leonid Mirny, Mikhail Gelfand<BR>
     Structural analysis of conserved base-pairs in protein-DNA complexes<BR>  
     &nbsp;<BR>
</LI>

<LI> Satoru Miyasaki, Hideaki Sugawara<BR>
     Visualization of features in flat file by use of DDBJ-XML<BR>  
     &nbsp;<BR>
</LI>

<LI> Michael Moorhouse, Milkos Cserzo<BR>
     An Alternative Approach to Investigating Sequence-Structure Relationships in Proteins<BR>  
     &nbsp;<BR>
</LI>

<LI> Tobias M&uuml;ller, Sven Rahmann, Rainer Spang, Martin Vingron<BR>
     TM: A Transmembrane-Specific Scoring Matrix<BR>  
     &nbsp;<BR>
</LI>

<LI> S.A.Mugilan, K. Veluraja<BR>
     SSPDP: A Novel method to Predict Secondary Structure from Amino Acid Sequences<BR>
     &nbsp;<BR>
</LI>

<LI> Rafael Najmanovich, Vladimir Sobolev, Marvin Edelman<BR>
     Population Boom: An Improved Genetic Algorithm<BR>  
     &nbsp;<BR>
</LI>

<LI> William R. Newell, Tara Acharya, J. Claiborne Stephens<BR>
     Statistical and Numerical Analysis of Genotypes and Haplotypes<BR>  
     &nbsp;<BR>
</LI>

<LI> Morten Nielsen, Claus Lundegaard, Thomas Nordahl Petersen, Jakob Bohr,
     S&ouml;ren Brunak, Garry P. Gippert, Ole Lund<BR>
     Novel method for template identification for globular proteins using
     sequence profiles and predicted secondary structure<BR>  
     &nbsp;<BR>
</LI>

<LI> J. P. Novak, R. Sladek, T. Hudson<BR>
     Variability in large-scale gene expression data<BR>  
     &nbsp;<BR>
</LI>

<LI> Humberto G. Ortiz-Zuazaga, Yolanda Robles, Ricardo Chiesa, SandraPe&ntilde;a de Ortiz<BR>
     Analysis of Learning-Related Changes in Gene Expression
     Using Nylon Membrane {cDNA} Microarrays<BR>  
     &nbsp;<BR>
</LI>

<LI> Lior S. Pachter, Jody Schwartz, Jim Lord, Alexander Fabrikant, Alexander Poliakov,
     Kelly A. Frazer, Inna Dubchak<BR>
     AVID and VISTA: Comparative Genomics Tools for Biological Discovery<BR>  
     &nbsp;<BR>
</LI>

<LI> Chris Pal, Mike Hu<BR>
     Methodologies for Constructing and Training Large Hierarchical Hidden Markov Models
     for Sequence Analysis<BR>  
     &nbsp;<BR>
</LI>

<LI> Ritu Pandey, David Selinger, David W. Mount, Vicki L. Chandler, Richard A. Jorgensen<BR>
     Comparative analysis of genes involved in regulating the chromatin environment in the genome
     of the model plant - Arabidopsis thaliana<BR>  
     &nbsp;<BR>
</LI>

<LI> E. M. Panina, A. A. Mironov, M. S. Gelfand<BR>
     New Fur-regulated Systems in Gamma-Proteobacteria<BR>  
     &nbsp;<BR>
</LI>

<LI> Gustavo Parisi, Juli&aacute;n Echave<BR>
     A model of structurally constrained protein evolution<BR>  
     &nbsp;<BR>
</LI>

<LI> Jean-Claude Perez<BR>
     Unifying Genomics and Proteomics in whole Genomes and Chromosomes<BR>  
     &nbsp;<BR>
</LI>

<LI> Jurgen Pletinckx, Antoine Janssen, Jan van Oeveren, Philippe Stas, Guus Simons,
     Ignace Lasters, Ren&eacute; van Schaik<BR>
     APES - Automated Processing and Extraction of Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Jurgen Pletinckx, Jan van Oeveren, Martin Reijans, Ren&eacute; Hogers,
     Philippe Stas, Michiel van Eijk, Ignace Lasters, Ren&eacute; van Schaik<BR>
     Software to Predict MicroArray Hybridization: ACROSS<BR>  
     &nbsp;<BR>
</LI>

<LI> Tal Pupko, Roded Sharan, Ron Shamir, Dan Graur<BR>
     A test for positive Darwinian selection<BR>  
     &nbsp;<BR>
</LI>

<LI> Krzysztof M. Rajkowski<BR>
     Improved Discrimination in Bayesian Profile Searching using Biconditional Residue
     Background Frequencies<BR>  
     &nbsp;<BR>
</LI>

<LI> Haluk Resat, John H. Miller, David A. Dixon, H. Steven Wiley<BR>
     Cellular Signal Regulation by Endocytic Trafficking<BR>  
     &nbsp;<BR>
</LI>

<LI> John A. Rose, Russell J. Deaton, Masami Hagiya, Akira Suyama<BR>
     The Fidelity of the Universal DNA Chip<BR>  
     &nbsp;<BR>
</LI>

<LI> Rickard Sandberg, G&ouml;sta Winberg, Ingemar Ernberg, Joakim C&ouml;ster<BR>
     Genome-wide identification of horizontal gene transfer events using Bayesian statistics<BR>  
     &nbsp;<BR>
</LI>

<LI> David Sankoff<BR>
     Short inversions and conserved gene clusters<BR>  
     &nbsp;<BR>
</LI>

<LI> Eran Segal, Ben Taskar, Audrey Gasch, Nir Friedman, Daphne Koller<BR>
     A Probabilistic Model for Generating Hypotheses using Gene Expression<BR>  
     &nbsp;<BR>
</LI>

<LI> Hwajung Seo, Haewon Nam, Ki-Bong Kim, Kiejung Park<BR>
     An Integrated Data Management System for Microbial Genome Projects<BR>  
     &nbsp;<BR>
</LI>

<LI> Felix B Sheinerman, Bissan Al-Lazikani, Barry Honig<BR>
     Predicting phosphopeptide selectivity of SH2 domains<BR>  
     &nbsp;<BR>
</LI>

<LI> Ronald Shigeta, Michael A. Siani-Rose, David Kulp<BR>
     RAKE: Accurate automated annotation of the human genome based on SCOP
     domain-derived hidden Markov models<BR>  
     &nbsp;<BR>
</LI>

<LI> Gregory A. C. Singer, Donal A. Hickey<BR>
     DNA Mutation Bias vs. Selection in Mitochondrial Proteins<BR>  
     &nbsp;<BR>
</LI>

<LI> James P. Sluka<BR>
     Leveraging Genomic Research with the Biomedical Literature<BR>  
     &nbsp;<BR>
</LI>

<LI> John Storey, David Siegmund<BR>
     Approximate P-values for Local Sequence Alignments: Numerical Studies<BR>  
     &nbsp;<BR>
</LI>

<LI> Fariza Tahi, Mireille Regnier, Manolo Gouy<BR>
     Automatic RNA secondary structure prediction with the comparative approach<BR>  
     &nbsp;<BR>
</LI>

<LI> Julie Thompson, Fr&eacute;d&eacute;ric Plewniak, Jean-Claude Thierry, Olivier Poch<BR>
     A new objective function for multiple sequence alignments<BR>  
     &nbsp;<BR>
</LI>

<LI> Michael Thorne, Marco Marra, Steven Jones<BR>
     Cataloguing Candidate Elements involved in Transcriptional Regulation in the 
<FONT FACE=italic>Caenorhabditis elegans</FONT> Genome<BR>  
     &nbsp;<BR>
</LI>

<LI> Pierre Tuff&eacute;ry, Pierre Darlu<BR>
     Is co-substitution a general process associated with the preservation
     of protein structure?<BR>  
     &nbsp;<BR>
</LI>

<LI> Mar&iacute;}a Elena V&aacute;zquez, F. Javier Arsuaga, Rainer K. Sachs,
     Philip Hahnfeldt, Lynn R. Hlatky<BR>
     Computer simulations suggest an early form of instability in
     irradiated human lymphocytes<BR>  
     &nbsp;<BR>
</LI>

<LI> Mar&iacute;a Silvina Fornasari, Juli&aacute;n Echave<BR>
     Comparison of nucleotide, codon, and amino acid substitution
     models in evolutionary distance estimation<BR>  
     &nbsp;<BR>
</LI>

<LI> Natalia Volfovsky, Steven L. Salzberg<BR>
     Clustering Method for Repeat Analysis of DNA Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Yufeng Wang, Jianying Gu, Xun Gu<BR>
     Exploring Functional Divergence from Sequence Evolution<BR>  
     &nbsp;<BR>
</LI>

<LI> Dunrui Wang, Victoria Yamazaki, Tom Tang<BR>
     Functional Classification of Proteins by Hierarchical Categorization of Pfam Models<BR>  
     &nbsp;<BR>
</LI>

<LI> Allison Waugh, Russ Altman, Patrick Gendron, Francois Major, James W. Brown,
     David Case, Daniel Gautheret, Robin Gutell, Stephen C. Harvey, Tom Macke,
     John Westbrook, Eric Westhof, Michael Zucker<BR>
     RNAML: An XML Format for RNA Data-Exchange<BR>  
     &nbsp;<BR>
</LI>

<LI> Min Wei, Daniel Duan, Chuanbo Xu<BR>
     Genetic Variation Study with Integrated KEGG Pathway Data<BR>  
     &nbsp;<BR>
</LI>

<LI> C. Nicole White, Ilya A. Vakser, E. Starr Hazard, Zhen Zhang<BR>
     Using Physicochemical Properties in Alignment of Multiple Protein Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Christoph K. Wierling, Matthias Steinfath, Ralf Herwig, Mario Drungowski,
     Thorsten Elge, Hans Lehrach<BR>
     Simulation and evaluation of complex hybridisation experiments<BR>  
     &nbsp;<BR>
</LI>

<LI> Shiquan Wu, Xun Gu<BR>
     A Greedy Algorithm for Optimal Recombination<BR>  
     &nbsp;<BR>
</LI>

<LI> Li Cheng Wu, Jorng-Tzong Horng<BR>
     Finding Tandem Repeats Online in Genome Sequences<BR>  
     &nbsp;<BR>
</LI>

<LI> Masao Yamaguchi, Emi Ichihashi, Yozo Ohnishi, Ryo Yamada, Toshihiro Tanaka,
     Yusuke Nakamura, Tatsuhiko Tsunoda<BR>
     Integration System of SNP Genotyping for Association Study in Millennium Project<BR>  
     &nbsp;<BR>
</LI>

<LI> Chen-Hsiang Yeang, Sridhar Ramaswamy, Pablo Tamayo, Jill P. Mesirov,
     Sayan Mukherjee, Ryan M. Rifkin, Todd Golub, Eric Lander<BR>
     Classifying multiple tumor types with gene expression data<BR>  
     &nbsp;<BR>
</LI>

<LI> Svetlana Zakharova, Johan R.C. van der Maarel<BR>
     Mesoscopic order in supercoiled DNA solutions<BR>  
     &nbsp;<BR>
</LI>

<LI> Svetlana Zakharova, Johan R.C. van der Maarela<BR>
     Multivalent Ion - DNA Interaction <BR>  
     &nbsp;<BR>
</LI>

<LI> Evgeni Zdobnov, Henning Hermjakob, Rolf Apweiler<BR>
     InterProScan - an integration tool for the signature-recognition methods in InterPro<BR>  
     &nbsp;<BR>
</LI>

<LI> Ming Zhang, Lydia E. Kavraki<BR>
     Efficiently Maintaining Molecular Conformations Using Local Frames<BR>  
     &nbsp;<BR>
</LI>


</OL>

<P>

<P><A HREF="Welcome.html">
<FONT FACE="HELVETICA" SIZE="3">-&gt; RECOMB 2001</FONT></A></P>

</FONT>
</UL>
</BODY>
</HTML>


